docs/source/database.rst

```diff
@@ -84,35 +84,32 @@ Following tables show columns according to different Categories:
+  |                        | product            | The name of the application requesting an update.| Exact string match only         | "Firefox" or "B2G"         |
   |                        +--------------------+--------------------------------------------------+---------------------------------+----------------------------+
-  |                        | systemCapabilities | The supported hardware features of the           | Simplified boolean string       | 	"SSE" or "MMX, SSE" or    |
-  |                        |                    | application requesting an update. This field     | match. '&&' ANDs terms while    |  "GenuineIntel && MMX,SSE" |
-  |                        |                    | is primarily used to point desupported users     | ',' ORs them. Terms are matched |                            |
-  |                        |                    | based on their hardware. Eg: users who do not    | using exact strings.            |                            |
+  |                        | instructionSet     | The most modern instruction set supported by the | Full string match or comma      | "SSE" or "MMX, SSE"        |
+  |                        |                    | client requesting an update. This field          | separated list of full strings  |                            |
+  |                        |                    | is primarily used to desupport users             | to match on                     |                            |
+  |                        |                    | based on their hardware. Eg: users who do not    |                                 |                            |
   |                        |                    | support SSE2                                     |                                 |                            |
   |                        +--------------------+--------------------------------------------------+---------------------------------+----------------------------+
-  |                        | Version            | The version of the application requesting an     | Exact string match or exact     | "36.0" or "36.0,36.1,36.2" |
+  |                        | memory             | The amount of RAM, in megabytes, that the client | Exact match or operator plus    | 8096, <8096, >=8096        |
+  |                        |                    | requesting the update has                        | memory to compare the incoming  |                            |
+  |                        |                    |                                                  | one against                     |                            |
+  |                        +--------------------+--------------------------------------------------+---------------------------------+----------------------------+
+  |                        | version            | The version of the application requesting an     | Exact string match or exact     | "36.0" or "36.0,36.1,36.2" |
```

Comment:
Strange, it looks like this cell ends up getting rendered as <dt> and <dd>, which has some special styling. I'll dig into it.